CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* [#12958](https://github.com/cosmos/cosmos-sdk/pull/12958) (deps) Downgrade to Tendermint [v0.34.21](https://github.com/tendermint/tendermint/releases/tag/v0.34.21).
```

Comment:
We never released anything with TM v0.35. So why the need for an entry?

Reply:
There is a CHANGELOG about TM v0.35 as well, so I think we should have one too for consistency, so if you read the CHANGELOG you understand it was upgraded and then downgraded, otherwise that's not clear.

Reply:
But it was never released....so they should both be removed.

Reply:
Alright, I can clean up the occurences 👍🏾